src/application.rs

```diff
@@ -41,6 +41,7 @@ pub fn create_app() -> Result<rocket::Rocket, Box<dyn Error>> {
         Err(err) => return Err(Box::new(err)),
     };
     let storage: Box<dyn Storage> = Box::new(SqlStorage::new(&database_url)?);
+    let token_validator: Box<dyn Validator> = Box::new(JwtValidator::from_env()?);
```

Comment:
nit: Is it possible to do something like this: `JwtValidator::from_config(app.config())`? Just to make sure our source of truth is an application config? Even more, I'd suggest putting these JWT related keys to our own `Config` struct by extracting them from `rocket.config()`, and pass that `Config` instance around.

Reply:
Good call! Sorry, I completely forgot about our desire to support TOML-based Rocket config! :(